okio/src/nonJvmMain/kotlin/okio/Sink.kt

```diff
@@ -15,12 +15,15 @@
  */
 package okio
 
-actual interface Sink {
+actual interface Sink : Closeable {
```

Comment:
I ended up having to put `Sink` back so the JVM one could keep the `Flushable` interface.